test/conformance/kuttl/validate/clusterpolicy/standard/enforce/ns-selector-with-wildcard-kind/05-delete-clulsterrole.yaml

```diff
@@ -0,0 +1,7 @@
+# Specifying the kind as `TestStep` performs certain behaviors like this delete operation.
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
```

Comment:
For some reason this causes CI to fail with:
```
    harness.go:390: more than 1 TestStep not allowed in step "delete-clulsterrole"
```
Maybe this `05-delete-ns.yaml` needs to be renamed to `06-delete-ns.yaml`

Reply:
Good catch!